x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/config_form.tsx

```diff
@@ -85,12 +78,7 @@ export const ConfigForm: React.FC<{
 
   return (
     <PolicyDetailCard>
-      <EuiCard
-        description={description}
-        data-test-subj={dataTestSubj}
-        textAlign="left"
-        title={typeTitle}
-      >
+      <EuiCard description data-test-subj={dataTestSubj} textAlign="left" title={typeTitle}>
```

Comment:
i guess we should remove description? i think i forgot to do it last time

Reply:
Umm, I tried in my own PR but it causes issues.

Reply:
oh wait, i think it's required for an EuiCard so you're good to good. nvm mind that comment!